src/OrchardCore/OrchardCore/Environment/Shell/ShellHost.cs

```diff
@@ -183,7 +183,7 @@ async Task CreateAndRegisterShellsAsync()
                     }
                     catch (Exception ex)
                     {
-                        _logger.LogError(ex, "A tenant could not be started '{TenantName}'", settings.Name);
+                        _logger.LogError(ex, "A tenant could not be started '{0}'", settings.Name);
```

Comment:
But this is the standard pattern that @sebastienros recommended.

Reply:
oh!

Reply:
You can learn about it here: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-2.1